src/api.rs

```diff
@@ -207,6 +208,22 @@ impl API {
         }
     }
 
+    pub fn scrape_expiries(&self) -> PbResult<Vec<String>> {
+        let client = reqwest::blocking::Client::new();
+        let response = client.get(self.base()).send()?;
```

Comment:
The reqwest client can be made with `preconfigured_privatebin_request_builder`. This pre-configures the client correctly with needed headers, e.g oidc auth headers. 

Since this request isn't a JSON request, the line inside the function `request = request.header("X-Requested-With", "JSONHttpRequest");` needs to be conditional.

Reply:
Indeed, I had tried using `preconfigured_privatebin_request_builder`, but then noticed the "JSONHttpRequest" was preventing html downloading. So did it this way. 

Do you suggest an addition of a parameter to `preconfigured_privatebin_request_builder` to make this  header conditional?

Reply:
Yea, thats what I was thinking. Just a simple isJsonRequest should do